roles/roles_common.js

```diff
@@ -1047,6 +1050,41 @@ Object.assign(Roles, {
     return false
   },
 
+  /**
+   * Check if current user is in at least one of the target roles.
+   * @method isInRole
+   * @param {String} role Name of role or comma-seperated list of roles.
+   * @param {String} [scope] Optional, name of scope to check.
+   * @return {Boolean} `true` if current user is in at least one of the target roles.
+   * @static
+   */
+  isInRole: function (role, scope) {
+    const userId = Meteor.userId()
+    const comma = (role || '').indexOf(',')
+    let roles
+
+    if (!userId) return false
+    if (!Match.test(role, String)) return false
+
+    if (comma !== -1) {
+      roles = role.split(',').reduce(function (memo, r) {
+        if (!r) {
+          return memo
+        }
+        memo.push(r)
+        return memo
+      }, [])
+    } else {
+      roles = [role]
+    }
+
+    if (Match.test(scope, String)) {
+      return this.userIsInRole(userId, roles, scope)
+    }
+
+    return this.userIsInRole(userId, roles)
```

Comment:
This could be a conditional return:

```js
return Match.test(scope, String)
  ? this.userIsInRole(userId, roles, scope)
  : this.userIsInRole(userId, roles)
```